src/migrations/migration.js

```diff
@@ -111,6 +114,13 @@ export default class Migration {
     while (true) {
       objects.push(...response.hits.hits);
       if (objects.length === this.getTotalHitCount(response)) {
+        if (response._scroll_id) {
```

Comment:
(Optional) That's a super weird looking loop. This might make more sense:
```
    while (objects.length < this.getTotalHitCount(response)) {
      objects.push(...response.hits.hits);
      response = await this._client.scroll({
        body: {
          scroll:    '1m',
          scroll_id: response._scroll_id
        }
      });
    }

    await this._client.clearScroll({
      body: {
        scroll_id: response._scroll_id
      }
    });

    return objects;
  }
```
_scroll_id seems to always be there if `scroll` present in the request, so the `if (response._scroll_id)` also seem pointless:
![image](https://user-images.githubusercontent.com/18154526/88556927-7a258980-d021-11ea-944b-5f13c05fb94b.png)